Earthfile

```diff
@@ -212,12 +212,16 @@ test-adapters:
     DO rust+SET_CACHE_MOUNTS_ENV
     ARG DELTA_TABLE_TEST_AWS_ACCESS_KEY_ID
     ARG DELTA_TABLE_TEST_AWS_SECRET_ACCESS_KEY
+    ARG ICEBERG_TEST_AWS_ACCESS_KEY_ID
+    ARG ICEBERG_TEST_AWS_SECRET_ACCESS_KEY
+    # Dependencies needed by the Iceberg test
+    RUN pip3 install -r crates/iceberg/src/test/requirements.ci.txt
```

Comment:
probably it's going to make a  mess but it would be nice to have these requirements in a global uv.lock file e.g., move https://github.com/feldera/feldera/blob/main/python/uv.lock to / and then just use that

Reply:
that would be nice, but I thought `uv.lock` is only the dependencies required by the Python SDK, i.e., things that anyone who uses the SDK needs to install, unlike these iceberg dependencies, which are only needed to run tests.